angular.json

```diff
@@ -127,6 +127,9 @@
               "devServerTarget": "oracc:serve:production"
             }
           }
+        },
+        "deploy": {
+          "builder": "angular-cli-ghpages:deploy"
```

Comment:
So this reads the git config to see where to push it and to authenticate, huh? Neat.